### ISSUE TITLE
Update composer to 2.0 - Notice of potential breaking change

### ISSUE THREAD

**Issue by stevenlafl** [DrupalStand/drupal-dev-docker#62]
mobomo/composer now has available the `2.0` tag. Please test this as `latest` will eventually become `2.0`

**Comment by estettler** [comment]
Does it work yet with Composer 2.0?

**Comment by stevenlafl** [comment]
@estettler Yes, I have migrated many DrupalStand projects successfully to 2.0.

Ahead of forcing that upgrade, you can edit this file https://github.com/DrupalStand/drupal-dev-docker/blob/master/bin/composer and add `:2.0` as the tag for the `mobomo/composer` docker image.

However you are right that we need to update `:latest` to use the `2.0` branch and force this upgrade eventually.

**Comment by estettler** [comment]
Excellent. Thanks for the quick response!
